### ISSUE TITLE
Suporte a Nginx e Php7

### ISSUE THREAD

**Issue by joelburigo** [prhost/new-opencart#17]
Acho importante em termos de performance e oferecer a oportunidade de fazer módulos já na nova e melhor versão do PHP

**Comment by prhost** [comment]
Sim, podemos disponibilizar os dois arquivos modelos, um do htaccess e o outro em nginx, ai o desenvolvedor escolhe qual usar.Precisamos de você nessa então. Desconheço nginx. Já sobre o php 7, como comentei, é interessante mantermos compatibilidade com a nova versão, assim melhora o desempenho e atrai mais colaboradores (:

**Comment by vilsongabriel** [comment]
Dar suporte ao nginx é bem tranquilo, já existem diversos modelos de configuração funcionais por ai, agora quanto ao PHP7 estou bem por fora das novidades.

**Comment by prhost** [comment]
@vilsongabriel Sobre o php7 acho que só precisaríamos adaptar o código para rodar, modificando algumas funções que foram depreciada e tal, precisaríamos de inicial pegar um opencart e rodar em php7 só pra ver o que acontece, alguém já fez isso?
